wallet/txauthor/author.go

```diff
@@ -309,6 +323,54 @@ func spendWitnessKeyHash(txIn *wire.TxIn, pkScript []byte,
 	return nil
 }
 
+// spendTaprootKeyHash generates, and sets a valid witness for spending the
```

Comment:
_technically_ there's no key hash here, as the main pubkey is used in the output script, and the witness is _just_ the signature (and not also the public key)